test/functional/fixtures/api/es-next/type/test.js

```diff
@@ -43,4 +44,20 @@ describe('[API] t.typeText()', function () {
                 expect(errs[0]).to.contains('> 19 |    await t.typeText(NaN, \'a\');');
             });
     });
+
+    if (!config.proxyless) {
+        it('Should not get selector the second time if the error was raised in the first.', function () {
```

Comment:
`Should not execute selector twice for non-existing element due to "confidential" option (GH-6623)`